pkg/apis/core/types.go

```diff
@@ -4814,7 +4835,7 @@ type ObjectReference struct {
 
 // LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
 type LocalObjectReference struct {
-	//TODO: Add other useful fields.  apiVersion, kind, uid?
+	// TODO: Add other useful fields.  apiVersion, kind, uid?
```

Comment:
Was this change gofmt induced?  Or accidental?

Reply:
Yeah this kind of changes is caused by the formatter in goland.